### PR TITLE
Add page - Create a simple vm from scratch

### PR DESCRIPTION
**Pull request by boulix3**
Related to #1266

### CODE REVIEW

File: docs/subnets/create-a-simple-vm-from-scratch.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+using the [Messenger](https://buf.build/ava-labs/avalanche/docs/main:messenger) service.
+Supposing that nodeY wants to build the block _you probably will implement some kind of background
+worker checking every second if there are any pending transactions_)_ :
+
+_on nodeY_
+
+- _client_ 
+[`Messenger.Notify`](https://buf.build/ava-labs/avalanche/docs/main:messenger#messenger.Messenger.Notify)
+  - You must issue a notify request to AvalancheGo by calling the method with the 
+  MESSAGE_BUILD_BLOCK value.
```
Comment (cgcardona) [inline docs/subnets/create-a-simple-vm-from-scratch.md:198]: Add backticks around `MESSAGE_BUILD_BLOCK`

**Review by cgcardona — CHANGES_REQUESTED** [review]
Thanks!

File: docs/subnets/create-a-simple-vm-from-scratch.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+  - To serve json-RPC endpoint, `/ext/bc/subnetId/rpc` exposed by AvalancheGo 
+  - See [json-RPC](#json-rpc) for more detail
+  - Create a [`Http`](https://buf.build/ava-labs/avalanche/docs/main:http) server and get its url.
+  - Return : a CreateHandlersResponse containing a single item with the server's url. (or an empty
+  array if not implementing the json-RPC endpoint)
+- [VM.StateSyncEnabled](https://buf.build/ava-labs/avalanche/docs/main:vm#vm.VM.StateSyncEnabled)
+  - Return : true if you want to enable StateSync, false otherwise.
+- [VM.SetState](https://buf.build/ava-labs/avalanche/docs/main:vm#vm.VM.SetState)
+    _If you had specified true in the StateSyncEnabled result_ 
+  - Param : a
```
[the remark this answers is not in the document]
Comment (boulix3) [reply docs/subnets/create-a-simple-vm-from-scratch.md:112]: The rest of the text is on the next line because of the line length restriction.

**Comment by boulix3** [comment]
Hello @cgcardona, @yulin-dong, I have a new page to add, but it's quite big and I don't know if I should add it to this PR.
[Create vm from scratch using c#](https://github.com/boulix3/avalanche-docs/blob/csharp-vm/docs/subnets/create-a-simple-vm-from-scratch-csharp.md)
Here is a [preview of the page](https://boulixavalanchedocs.web.app/subnets/create-a-simple-vm-from-scratch-csharp) hosted on firebase.
What should I do?
- Add it to this PR
- Wait for this PR to be completed and create a new one
- Create a parallel PR (but containing changes relating to this one as well)

**Comment by yulin-dong** [comment]
> Hello @cgcardona, @yulin-dong, I have a new page to add, but it's quite big and I don't know if I should add it to this PR. [Create vm from scratch using c#](https://github.com/boulix3/avalanche-docs/blob/csharp-vm/docs/subnets/create-a-simple-vm-from-scratch-csharp.md) Here is a [preview of the page](https://boulixavalanchedocs.web.app/subnets/create-a-simple-vm-from-scratch-csharp) hosted on firebase. What should I do?
> 
> * Add it to this PR
> * Wait for this PR to be completed and create a new one
> * Create a parallel PR (but containing changes relating to this one as well)

Please create a separate PR. Thanks.

**Review by boulix3 — COMMENTED** [review]
Applied all requested changes

**Comment by cgcardona** [comment]
Closing in favor of https://github.com/ava-labs/avalanche-docs/pull/1310

**Pull request closed without merging**